RELEASE-NOTES.md

```diff
@@ -2,10 +2,48 @@
 
 ## Release 2.14.5
 
-### Highlight TBD
+### Storage block compatibility with `flow.from_source`
+
+You can now use all your existing storage blocks with `flow.from_source`! Using storage blocks with `from_source` is great when you need to synchronize your credentials and configuration for your code storage location with your flow run execution environments. Plus, because block configuration is stored serverside and pulled at execution time, you can update your code storage credentials and configuration without redeploying your flows!
```

Comment:
```suggestion
You can now use all your existing storage blocks with `flow.from_source`! Using storage blocks with `from_source` is great when you need to synchronize your credentials and configuration for your code storage location with your flow run execution environments. Plus, because block configuration is stored server-side and pulled at execution time, you can update your code storage credentials and configuration without re-deploying your flows!
```